plugins/inputs/windows_event_log/wineventlog/utils_test.go

```diff
@@ -46,15 +46,15 @@ func TestDecodingWithDifferentBufferUsedReturned(t *testing.T) {
 	data, _ := hex.DecodeString(originalHexStr)
-	bufferUsed := len(data)/2
+	bufferUsed := len(data) / 2
```

Comment:
Why is go fmt inconsistent when doing operations? I noticed that when formatting, it removes spaces around `+` but here, it added spaces around `/`